style.css

```diff
@@ -0,0 +1,196 @@
+body {
+    height: 10%;
+    width: 45%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#Resume{
+    display: flex;
+    flex-direction: column;
+}
+
+#Resume div{
+    position: relative;
+}
+
+#Resume-title .outer-box
+{
+    border: 50px cadetblue solid;
+}
+
+#Resume-title .outer-box p{
+    text-align: center;
+    font-style: italic;
+    font-weight: bold;
+}
+
+.inner-box
+{
+    border: solid black;
+    margin: 1%;
+}
+
+#profile
+{
+    display: flex;
+    flex-direction: row;
+}
+
+.profile-info
+{
+   width: 33%;
+}
+.profile-picture
+{
+    width: 33%;
```

Comment:
reduce the size(width) of the window(browser) to find that contents of "profile" overlap, try 'width: -webkit-fill-available;' for the image(not the div) and see if it still overlaps.

Reply:
done